backend-tests/tests/conftest.py

```diff
@@ -65,3 +74,42 @@ def _get_endpoint_url(url):
     finally:
         for p, _ in processes.values():
             p.terminate()
+
+
+def create_tenant_test_setup() -> Tenant:
+    """ Creates a tenant, and a user belonging to the tenant. """
+    uuidv4 = str(uuid.uuid4())
+    tenant, username, password = (
+        "test.mender.io-" + uuidv4,
+        "some.user+" + uuidv4 + "@example.com",
+        "secretsecret",
+    )
+    tenant = create_org(tenant, username, password, "enterprise")
+    user = create_user(
+        "foo+" + uuidv4 + "@user.com", "correcthorsebatterystaple", tid=tenant.id
+    )
+
+    response = ApiClient(useradm.URL_MGMT).call(
+        "POST", useradm.URL_LOGIN, auth=(user.name, user.pwd)
+    )
+    assert response.status_code == 200
+    user.utoken = response.text
+    tenant.users = [user]
+    return tenant
+
+
+def create_user_test_setup() -> User:
+    """Create a user, log user in. """
+    uuidv4 = str(uuid.uuid4())
+    user_name, password = (
+        "some.user+" + uuidv4 + "@example.com",
+        "secretsecret",
+    )
+
+    user = create_user(user_name, password)
+    useradmm = ApiClient(useradm.URL_MGMT)
+    # log in user
+    response = useradmm.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
+    assert response.status_code == 200
+    user.utoken = response.text
+    return user
```

Comment:
In my opinion, these helpers should be in `testutils.common` and not in `conftest.py`. This file is meant to have only pytest fixtures, and it is included always when launching `pytest`.

Reply:
You are right, I moved helper functions to `testutils.common`. Thank you!